testing/populate_rest.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+///////////////////////////////////////////////////
+// This script is intended to be run as a cron job
+// every n-minutes or whatever the equiv is on windows
+//
+// Author: John A. [euclidjda](https://github.com/euclidjda)
```

Comment:
we don't have author name on any files, shouldn't start now, maybe port and mention source in commit message?